webui_lite/index.html

```
@@ -1510,9 +1514,12 @@ <h1>OpenFilamentSensor</h1>
                 </div>
                 <div style="padding: 16px;">
                     <div class="form-group checkbox-group">
-                        <input type="checkbox" id="enable_minimal_mode" ${isMinimalModeEnabled ? 'checked' : ''}>
-                        <label class="form-label" for="enable_minimal_mode" style="margin: 0;">Enable Minimal Mode</label>
-                        <p class="form-help" style="margin-left: 36px; margin-top: 8px;">Switch to a lightweight text-only interface. Disables charts, animations, and SSE streaming to reduce CPU/memory usage on low-power devices. This preference is saved in your browser.</p>
+                        <input type="checkbox" id="enable_minimal_mode" ${isMinimalModeEnabled ? 'checked' : '' }>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)